Adrienne / Fix token undefined during account switching session storage handling

adrienne-deriv opened the pull request:
## Changes:

Please provide a summary of the change.

### Screenshots:

Please provide some screenshots of the change.

adrienne-deriv commented on packages/core/src/Modules/Callback/CallbackPage.tsx:
```diff
@@ -13,6 +13,7 @@ const CallbackPage = () => {
                 localStorage.setItem('config.tokens', JSON.stringify(tokens));
                 localStorage.setItem('config.account1', tokens.token1);
                 localStorage.setItem('active_loginid', tokens.acct1);
+                sessionStorage.setItem('active_loginid', tokens.acct1);
```
Since after user logs out, there is no `tradershub_redirect_to` session storage, thus the if condition `if (redirectTo)` will never execute and `active_loginid` session storage is null. We just set a default value here

<img width="485" alt="Screenshot 2025-03-13 at 9 38 13 AM" src="https://github.com/user-attachments/assets/643a7f9c-b229-44bc-b53d-804f16bafcc6" />

adrienne-deriv replied:
https://github.com/user-attachments/assets/e392447a-0740-40c3-b4a7-cf51d3533219

amam-deriv reviewed (dismissed):
can explain to me a bit on why we need to update callback page there?

adrienne-deriv commented:
> can explain to me a bit on why we need to update callback page there?

Previously we've made this change to the client store, because this function `storeClientAccounts` was always overriding and defaulting our currency switching to `USD` (and not using our sessionStorage's currency), so we have added a default value here from the sessionStorage. This works fine for flows where we go from low-code TH -> `/redirect` page **because it will automatically set and populate the session Storage `active_loginid` value**
<img width="596" alt="Screenshot 2025-03-13 at 1 34 28 PM" src="https://github.com/user-attachments/assets/52d6e5f1-cdfa-4f8e-8423-4ab6699e8b96" />

<img width="801" alt="Screenshot 2025-03-13 at 1 28 56 PM" src="https://github.com/user-attachments/assets/8503f3b6-895a-4df0-bf17-c85262fca3f7" />

But for normal flows where we don't pass through the `/redirect` page, the session Storage `active_loginid` will be `null`, and this caused an issue within client-store where `active_loginid` is null and that error is thrown:
<img width="586" alt="Screenshot 2025-03-13 at 1 32 00 PM" src="https://github.com/user-attachments/assets/3c2e5858-9b7e-4fe3-afc0-ddc80c3d91e4" />

<img width="456" alt="Screenshot 2025-03-13 at 9 33 42 AM" src="https://github.com/user-attachments/assets/342752af-5975-4a80-a2b8-22eb38d3de27" />

So we added a default fallback for `active_loginid` here in session storage:
<img width="624" alt="Screenshot 2025-03-13 at 1 32 34 PM" src="https://github.com/user-attachments/assets/7ebf1f52-8a5b-4724-a55c-56bacb76bbad" />

And one more thing, we have an inconsistency between the session storage's `active_loginid` as well as the local storage's `active_loginid` (where by default, local storage's `active_loginid` is the first account `acct1`). I decided to sync them up to avoid any future issues where client store is logging in with a different `active_loginid` than what we have currently switched in the session storage's `active_loginid`
